statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -743,6 +743,16 @@ def forecast_interval(self, y, steps, alpha=0.05):
 
         return point_forecast, forc_lower, forc_upper
 
+    def to_vecm(self):
```

Comment:
That's already implemented in vecm.py (method var_repr of the VARResults class).